src/Graph.cpp

```diff
@@ -135,7 +137,7 @@ void Graph::serializeShadow(Graph& G)
             return;
         }
 
-        string outName = "networks/" + name + "/autogenerated/" + name + "_NodeNameToIndexMap" + ".bin";
+        string outName = "autogenerated/serialized/" + name + "_NodeNameToIndexMap" + ".bin";
```

Comment:
Ditto, ditto, ditto. One place. (eg what if I decide at some point I want it to be /tmp/autogenerated? Now I need to change it in a million places.)